### PR TITLE
chore(products/repositories): adapt/update whole list of products/repositories

### PR DESCRIPTION
**Pull request by stephanbcbauer**
## Description

The main purpose for this PR is to update our product list. It's old and not up to date. The initial idea was to publish all our active products in this list. Since we are always asking about the responsible committer for each repository, I tried to update the list for each repository depending on the committers' contribution to the repository.

What is needed in this PR @eclipse-tractusx/automotive-tractusx-committers, please have a look at your product, repositories or where you are listed as committer. Is this correct? Is there a description or something missing within your product? Should we describe something more in detail? Would be happy about your sugestions.

## Goal

If this PR is merged (hopefully we can do this) we would have a list of all products/repositories and the related committer which is responsible for this product/repository.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [X] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [X] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: utils/products.js
Comment (tom-rm-meyer-ISST) [inline utils/products.js:408]: Please remove the backend and frontend repository as those are archived.

From Committers, please remove @tomaszbarwicki. @mhellmeier for you it's OK to be listed here, isn't it?

File: utils/products.js
Comment (tom-rm-meyer-ISST) [inline utils/products.js:371]: Please remove me from here

**Review by tom-rm-meyer-ISST — CHANGES_REQUESTED** [review]
reviewed for puris / myself

File: utils/products.js
Comment (evegufy) [reply utils/products.js:371]: policy hub was already correctly added, please don't remove it: https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/968/files#diff-5c39f3e132d6486132f6c3b616116ec70bb0c8a661a364316f6cbcc299a5c3eeL138

File: utils/products.js
Comment (evegufy) [inline utils/products.js:3]: not a product, please don't add as product on website

File: utils/products.js
Comment (evegufy) [inline utils/products.js:16]: not a product, please don't add as product on website

File: utils/products.js
Comment (evegufy) [inline utils/products.js:41]: not a product, please don't add as product on website

File: utils/products.js
Comment (evegufy) [inline utils/products.js:62]: not a product, please don't add as product on website

File: utils/products.js
Comment (evegufy) [inline utils/products.js:102]: Please remove me

File: utils/products.js
Comment (evegufy) [inline utils/products.js:110]: not a product, please don't add as product on website

File: utils/products.js
Comment (evegufy) [inline utils/products.js:122]: not a product, please don't add as product on website

File: utils/products.js
Comment (evegufy) [inline utils/products.js:160]: not a product, please don't add as product on website

File: utils/products.js
Comment (evegufy) [inline utils/products.js:179]: not a product, please don't add as product on website

File: utils/products.js
Comment (evegufy) [inline utils/products.js:430]: not a product, please don't add as product on website

File: utils/products.js
Comment (evegufy) [inline utils/products.js:446]: not a product, please don't add as product on website

File: utils/products.js
Comment (evegufy) [inline utils/products.js:662]: please don't add, it is to be archived

File: utils/products.js
Comment (evegufy) [inline utils/products.js:516]: please also add https://github.com/ntruchsess

File: utils/products.js
Comment (evegufy) [inline utils/products.js:506]: please add https://github.com/ntruchsess

**Review by evegufy — CHANGES_REQUESTED** [review]

File: utils/products.js
Comment (matbmoser) [inline utils/products.js:205]: I will delete this product, this repo is archived and was moved to the main webpage of tractus-x

**Comment by matbmoser** [comment]
Ok, I have updated our Digital Product Pass product description and configurations. I also removed the eco pass kit repo, since it is not really a product and it is archived.

**Review by matbmoser — APPROVED** [review]
At least from my side I can approve the merge.

File: utils/products.js
Comment (SebastianBezold) [inline utils/products.js:444]: Similar to the other "non-products", that @evegufy mentioned, I also wouldn't add the Security SIG.
If committer info for SIGs is valuable, consider adding them in the "Communities / SIGs" pages: https://eclipse-tractusx.github.io/community/sigs/intro

File: utils/products.js
Comment (SebastianBezold) [inline utils/products.js:551]: Does that also qualify as "non-product" and should therefore also be removed?

File: utils/products.js
Comment (SebastianBezold) [inline utils/products.js:612]: Has this list been generated by auomation? If so, I would consider adding the `--no-archive` flag. 
This is only an enabling automation repo and not a product. It is also archived.
If you want to keep it anyways, please remove me from the list of committers

**Review by SebastianBezold — CHANGES_REQUESTED** [review]
If this list was generated by a script or similar, it should maybe be re-run without taking archived repos into account. 
I already suggested in the past, that if this list is maintained here, we should maybe automate it with the same logic as the TRG dashboard is generated: https://eclipse-tractusx.github.io/sig-release/
This dashboard already groups repos to products and also ignores archived repos. SIGs and supporting repos are also already separated. 
Maybe both approaches (website + dashboard) can be combined, so that there is no duplication and manual effort.

@almadigabor and @tomaszbarwicki: Could you two please double check this PR explicitly? You are referenced in a lot of product, where I guess you did TRG checks or merged contributions. I guess you are not actually involved in all of the products

File: utils/products.js
Comment (mhellmeier) [inline utils/products.js:268]: Are EDC extensions a specific product?

File: utils/products.js
Comment (mhellmeier) [reply utils/products.js:359]: I think @tom-rm-meyer-ISST only wants to remove his name from the list instead of the whole product.

File: utils/products.js
Comment (mhellmeier) [inline utils/products.js:511]: Documentation is also not a product.

File: utils/products.js
Comment (mhellmeier) [inline utils/products.js:535]:
```suggestion
      "The project provides a business application for tracking parts along the supply chain. It uses quality notifications in a standardized way to notify customers or suppliers about detected faulty parts. It is based on the Catena-X standards for serialized and non-serialized hardware and software components.",
```

File: utils/products.js
Comment (mhellmeier) [inline utils/products.js:592]: @jimmarino is missing in the list.

**Review by mhellmeier — CHANGES_REQUESTED** [review]

File: utils/products.js
Comment (SujitMBRDI) [inline utils/products.js:657]: If we are keeping this product, please remove me from here

File: utils/products.js
Comment (SujitMBRDI) [inline utils/products.js:83]: Fabio is no active committer now, can remove from here

File: utils/products.js
Comment (tom-rm-meyer-ISST) [reply utils/products.js:359]: True. I only would add the current maintainers to the list. I just contributed a small pr :)

File: utils/products.js
Comment (almadigabor) [reply utils/products.js:268]: I'd say this is also part of product "Knowledge Agents". I would just add the repo url to the product above. What do you think @drcgjung?

File: utils/products.js
Comment (almadigabor) [inline utils/products.js:307]: `managed-simple-data-exchanger`, `managed-simple-data-exchanger-backend`, `managed-simple-data-exchanger-frontend` are the same product, just different components.

File: utils/products.js
Comment (almadigabor) [inline utils/products.js:565]: Shall we treat the umbrella chart as a product?

**Comment by almadigabor** [comment]
> If this list was generated by a script or similar, it should maybe be re-run without taking archived repos into account. I already suggested in the past, that if this list is maintained here, we should maybe automate it with the same logic as the TRG dashboard is generated: https://eclipse-tractusx.github.io/sig-release/ This dashboard already groups repos to products and also ignores archived repos. SIGs and supporting repos are also already separated. Maybe both approaches (website + dashboard) can be combined, so that there is no duplication and manual effort.
> 
> @almadigabor and @tomaszbarwicki: Could you two please double check this PR explicitly? You are referenced in a lot of product, where I guess you did TRG checks or merged contributions. I guess you are not actually involved in all of the products

Hey @SebastianBezold, I had a look at the products where I'm listed as committer and added comments where I see fit.

**Comment by stephanbcbauer** [comment]
Thank you all for your review and feedback. Hopefully, I changed everything like expected. @almadigabor @tomaszbarwicki on your side, I am still not sure, please have a look again. Thank you very much.

File: utils/products.js
Comment (tom-rm-meyer-ISST) [inline utils/products.js:87]: Aren't the daps and its registration service ready for deprecation?

File: utils/products.js
Comment (tom-rm-meyer-ISST) [inline utils/products.js:135]: committers available, thus product description MUST be set

File: utils/products.js
Comment (tom-rm-meyer-ISST) [inline utils/products.js:148]: committers available, thus product description MUST be set

File: utils/products.js
Comment (tom-rm-meyer-ISST) [inline utils/products.js:163]: committers available, thus product description MUST be set

File: utils/products.js
Comment (tom-rm-meyer-ISST) [inline utils/products.js:241]: Where does this project come from? It has been a team and this is a summary of all products belonging to the team. 

What's this list's target? If it's about listing products and repositories I would expect them to be separated because e.g. an adopter does not need to use all of these services to have one application runnint (e.g. frontend, backend, documentation, etc. for portal). Then only the leading repository of the  `.tractusx` needs to be listed.

File: utils/products.js
Comment (tom-rm-meyer-ISST) [inline utils/products.js:265]: committers available, thus product description MUST be set

File: utils/products.js
Comment (tom-rm-meyer-ISST) [inline utils/products.js:365]:
```suggestion
    productDescription: "The Catena-X Country Risk project is a web application that calculates a risk score per country based on information regarding corruption, political stability, economic risk, and social and structural figures.",
```

File: utils/products.js
Comment (tom-rm-meyer-ISST) [inline utils/products.js:4]:
```suggestion
    productDescription: "Welcome to the API Hub repository, a centralized location for hosting and viewing API documentation for the Tractus-X organization. This repository automates the collection of OpenAPI specifications from GitHub releases, generates Swagger UI documentation, and publishes it on GitHub Pages.",
```

File: utils/products.js
Comment (tom-rm-meyer-ISST) [inline utils/products.js:69]: This product forwards to another [github project & repo](eclipse-dataspace-dcp/decentralized-claims-protocol), likely as this has been moved to be handled by the EF working groups more independent of Eclipse Tractus-X.

Recommend removing

File: utils/products.js
Comment (tom-rm-meyer-ISST) [inline utils/products.js:352]:
```suggestion
    productDescription: "Defines Credential and Policy Profiles for Tractus-X.",
```

File: utils/products.js
Comment (tom-rm-meyer-ISST) [inline utils/products.js:232]: product not available

**Review by tom-rm-meyer-ISST — CHANGES_REQUESTED** [review]
Walked through in more detail. I proposed `productDescription` for central or non-committer repos. 

Also there are non-existing repos listed, I would remove or split.

File: utils/products.js
Comment (evegufy) [reply utils/products.js:69]: Yes, should be removed, https://github.com/eclipse-tractusx/identity-trust moved to https://github.com/eclipse-dataspace-dcp/decentralized-claims-protocol

File: utils/products.js
Comment (evegufy) [inline utils/products.js:88]: the two repos should be removed and the product be changed to Self-Description Factory.

File: utils/products.js
Comment (evegufy) [reply utils/products.js:232]: I assume the [app-dashboard](https://github.com/eclipse-tractusx/app-dashboard) but that's not a product

**Comment by stephanbcbauer** [comment]
> Walked through in more detail. I proposed `productDescription` for central or non-committer repos. 
> 
> 
> 
> Also there are non-existing repos listed, I would remove or split.

Will have a look. Thx @tom-rm-meyer-ISST

**Review by evegufy — COMMENTED** [review]
could you please also update the PR description, for instance the following statement is obsolete:
>If there wasn't an active committer, I put @evegufy and myself as responsible committer. Of course, we can adapt this. I also deleted the committer, where we already know they are not present after the end of consortium.

**Review by matbmoser — DISMISSED** [review]

**Comment by matbmoser** [comment]
LGTM we should merge this ASAP, in worst case another PR should be created

**Comment by tunacicek** [comment]
sldt topics LGTM

**Comment by stephanbcbauer** [comment]
> sldt topics LGTM

@tunacicek THX, just to confirm, also this [comment](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/968#discussion_r1776395645)?

The SLDT Topic was the only open Part. So if it is OK for you, I would be happy for a approval :) Maybe also from your side @matbmoser

**Comment by tunacicek** [comment]
> > sldt topics LGTM
> 
> @tunacicek THX, just to confirm, also this [comment](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/968#discussion_r1776395645)?
> 
> The SLDT Topic was the only open Part. So if it is OK for you, I would be happy for a approval :) Maybe also from your side @matbmoser

@stephanbcbauer : Thanks for the feedback. I will take a look. Maybe it make sense as tom mention to separate the products in "semantic-layer & digital twin" and discovery services.

**Comment by stephanbcbauer** [comment]
> > > sldt topics LGTM
> > 
> > 
> > @tunacicek THX, just to confirm, also this [comment](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/968#discussion_r1776395645)?
> > The SLDT Topic was the only open Part. So if it is OK for you, I would be happy for a approval :) Maybe also from your side @matbmoser
> 
> @stephanbcbauer : Thanks for the feedback. I will take a look. Maybe it make sense as tom mention to separate the products in "semantic-layer & digital twin" and discovery services.

THX @tunacicek I will wait for feedback? Correct?

**Comment by tunacicek** [comment]
> > > > sldt topics LGTM
> > > 
> > > 
> > > @tunacicek THX, just to confirm, also this [comment](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/968#discussion_r1776395645)?
> > > The SLDT Topic was the only open Part. So if it is OK for you, I would be happy for a approval :) Maybe also from your side @matbmoser
> > 
> > 
> > @stephanbcbauer : Thanks for the feedback. I will take a look. Maybe it make sense as tom mention to separate the products in "semantic-layer & digital twin" and discovery services.
> 
> THX @tunacicek I will wait for feedback? Correct?

I separated the products into digital twin registry, semantic layer, and discovery services.
I hope this is fine.

**Review by matbmoser — APPROVED** [review]

**Comment by stephanbcbauer** [comment]
@mhellmeier , @tom-rm-meyer-ISST you requested changes, we already adapted them. Is it OK for you to merge the PR for now? Would like to work on a clean state with beginning of the new year

**Review by mhellmeier — APPROVED** [review]
Thanks for implementing the changes!

**Review by tunacicek — APPROVED** [review]

File: utils/products.js
Comment (tom-rm-meyer-ISST) [inline utils/products.js:81]:
```suggestion
```

remove `identity-trust` as discussed [here](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/968/files#r1776410697)

File: utils/products.js
Comment (tom-rm-meyer-ISST) [reply utils/products.js]: This has been split. Still not sure if it's perfect, but good to go.

**Review by tom-rm-meyer-ISST — CHANGES_REQUESTED** [review]
Hey, we missed to remove the `identity-trust` repo following my comments.

sidenote on the open collaboration: It's always good to leave a short notice on the comment like "done" if done. I resolved my comments and checcked on my own what's done.

**Comment by stephanbcbauer** [comment]
> Hey, we missed to remove the `identity-trust` repo following my comments.
> 
> sidenote on the open collaboration: It's always good to leave a short notice on the comment like "done" if done. I resolved my comments and checcked on my own what's done.

Thx, normally i add a done, maybe i missed it somewhere since there were so many changes

**Review by tom-rm-meyer-ISST — APPROVED** [review]
LGTM

**Comment by stephanbcbauer** [comment]
Thank you, everybody … let's rework the list if needed.

**Pull request merged**
